tests-integration/tests/http.rs

```diff
@@ -336,6 +336,20 @@ pub async fn test_prom_http_api(store_type: StorageType) {
         .await;
     assert_eq!(res.status(), StatusCode::OK);
 
+    // series
+    let res = client
+        .get("/api/v1/series?match[]=demo")
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .send()
+        .await;
+    assert_eq!(res.status(), StatusCode::OK);
+    let res = client
+        .post("/api/v1/series?match[]=up&match[]=down")
+        .header("Content-Type", "application/x-www-form-urlencoded")
+        .send()
+        .await;
+    assert_eq!(res.status(), StatusCode::OK);
```

Comment:
Can we assert if the response body is expected?

Reply:
By the way, can I also add tests for `/api/v1/labels` and `/api/v1/label/:label_name/values`?

Reply:
It seems like there aren't the response body tests for `/api/v1/labels` and `/api/v1/label/:label_name/values`

https://github.com/GreptimeTeam/greptimedb/blob/3dc45f1c138a926c70649021bc223b57f1766fba/tests-integration/tests/http.rs#L289-L371

Reply:
Good catch! Appreciate it if you can add them too. Thanks a lot.